e2e/utils/waits-utils.ts

```diff
@@ -392,29 +393,31 @@ export async function waitWhileLoading(
   await Promise.race([page.waitForSelector(notBlankPageSelector), page.waitForSelector(spinElementsSelector)]);
 
   // Wait for spinners stop and gone.
-  await Promise.all([
-    page.waitForFunction(
-      (css) => {
-        return !document.querySelectorAll(css).length;
-      },
-      { polling: 'mutation', timeout },
-      spinElementsSelector
-    ),
-    page.waitForSelector(spinElementsSelector, { hidden: true, timeout })
-  ]).catch((err: Error) => {
-    logger.error(`Failed wait for spinner stop: xpath="${spinElementsSelector}"`);
+  try {
```

Comment:
Update to use `try/catch` instead promise `.catch` to catch asyn and non-asyn errors.